src/__tests__/index.test.tsx

```diff
@@ -40,4 +42,19 @@ describe('react-native-piwik-pro-sdk', () => {
       );
     });
   });
+
+  describe('setCustomDimension', () => {
+    it('Set custom dimension', async () => {
```

Comment:
Either use `it` in a sentence form (i.e. "it sets custom dimension") or `test` in current form (i.e. "test set custom dimension").